dune-project

```diff
@@ -35,16 +35,13 @@
    (>= "5.1"))
   (cmdliner
    (>= "1.1.0"))
-  (base64
-   (>= "3.1.0"))
   dune-build-info
   (cppo :build)
   (ounit :with-test)
   (reason
    (and
     :with-test
     (>= "3.9.0")))
-  (js_of_ocaml :with-test)
```

Comment:
Isn't this needed for the playground tests?

https://github.com/melange-re/melange/blob/main/test/blackbox-tests/melange-playground/playground.t

Reply:
never mind, i just see it's a dep of `melange-playground` pkg.